### PR TITLE
[stable/spring-cloud-data-flow] Move chart to distributed bitnami repository

### PR DESCRIPTION
**Pull request by donbeave**
#### What this PR does / why we need it:

Given the [`stable` deprecation timeline](https://github.com/helm/charts#deprecation-timeline), the Bitnami maintained Spring Cloud Data Flow Helm chart is now located at [bitnami/charts](https://github.com/bitnami/charts/).

Please, use [this issue](https://github.com/helm/charts/issues/20969) for common questions about the migration/deprecation process, for PR/Issues related to the chart itself, please visit the [bitnami/charts](https://github.com/bitnami/charts/) GitHub repository.

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/spring-cloud-data-flow/Chart.yaml [hunk 1]
```diff
@@ -1,17 +1,13 @@
 apiVersion: v1
-description: Toolkit for building data processing pipelines.
+description: DEPRECATED Toolkit for building data processing pipelines.
 name: spring-cloud-data-flow
-version: 2.7.1
+version: 2.7.2
 appVersion: 2.5.1.RELEASE
+# The redis chart is deprecated and no longer maintained. For details deprecation, see the PROCESSES.md file.
```
Comment (mattfarina) [inline stable/spring-cloud-data-flow/Chart.yaml]: I'm guessing that this was a copy and paste error. Did you mean to say redis?

Comment (donbeave) [reply stable/spring-cloud-data-flow/Chart.yaml]: Yeah, fixed.

**Comment by chrisjs** [comment]
@donbeave sorry for the delay, we are ready to move forward on this. if you would like to update the conflict we can push this going again

**Comment by donbeave** [comment]
@chrisjs conflicts resolved

File: stable/spring-cloud-data-flow/README.md [hunk 2]
```diff
@@ -4,6 +4,27 @@
 
 Data processing pipelines consist of [Spring Boot](https://projects.spring.io/spring-boot/) apps, built using the [Spring Cloud Stream](https://cloud.spring.io/spring-cloud-stream/) or [Spring Cloud Task](https://cloud.spring.io/spring-cloud-task/) microservice frameworks. This makes Spring Cloud Data Flow suitable for a range of data processing use cases, from import/export to event streaming and predictive analytics.
 
+## This Helm chart is deprecated
+
+Given the [`stable` deprecation timeline](https://github.com/helm/charts#deprecation-timeline), the Bitnami maintained Redis Helm chart is now located at [bitnami/charts](https://github.com/bitnami/charts/).
```
Comment (chrisjs) [inline stable/spring-cloud-data-flow/README.md]: `Redis` -> `Spring Cloud Data Flow`

File: stable/spring-cloud-data-flow/templates/NOTES.txt [hunk 3]
```diff
@@ -1,3 +1,23 @@
+This Helm chart is deprecated
+
+Given the `stable` deprecation timeline (https://github.com/helm/charts#deprecation-timeline), the Bitnami maintained Redis Helm chart is now located at bitnami/charts (https://github.com/bitnami/charts/).
```
Comment (chrisjs) [inline stable/spring-cloud-data-flow/templates/NOTES.txt]: `Redis` -> `Spring Cloud Data Flow`

**Review by chrisjs — CHANGES_REQUESTED** [review]

File: stable/spring-cloud-data-flow/README.md (hunk 2 above)
Comment (donbeave) [reply stable/spring-cloud-data-flow/README.md]: Oops, sorry for this. Fixed.

File: stable/spring-cloud-data-flow/templates/NOTES.txt (hunk 3 above)
Comment (donbeave) [reply stable/spring-cloud-data-flow/templates/NOTES.txt]: Fixed.

**Pull request merged**